components/src/core/Drawer/Drawer.tsx

```diff
@@ -285,16 +261,11 @@ DrawerComponent.propTypes = {
         root: PropTypes.string,
         content: PropTypes.string,
         paper: PropTypes.string,
-        paperMobile: PropTypes.string,
-        smooth: PropTypes.string,
     }),
     open: PropTypes.bool.isRequired,
     width: PropTypes.number,
     ...PXBlueDrawerNavGroupInheritablePropertiesPropTypes,
 };
-DrawerComponent.defaultProps = {
-    variant: 'permanent',
-};
 DrawerComponent.defaultProps = {
```

Comment:
default variant = persistent?

Reply:
This default thing will override the default variant we provided in the drawerLayout, so I'd have to take them out.

Reply:
Let me do that inside the Drawer component...

Reply:
> > Maybe we can see if it's a drawer, and if not, see if the first child is a drawer and clone that like evan suggests?
> 
> Does this drawer check have to be limited to a `PXBlueDrawer` or including any thing with a `Drawer` displayName?

Probably a PX Blue drawer only...Our DrawerLayout is not going to work with a MUI drawer (ask @emclaug2 about all the trouble he had with the showcase trying to do that by accident)